ProcessMaker/processmaker
# FOUR-13162: The categories are not listed alphabetically in Launch Pad

## Issue & Reproduction Steps
The categories are not listed alphabetically in Launch Pad 

## How to Test

1. Create categories related to the process
2. Go to Process catalogue
3. See the list of categories
4. 

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13162

## Code Review Checklist
ci:deploy
ci:next
